frontend/src/bundles/talent-onboarding/components/skills-step/components/skills-project-links.tsx

```diff
@@ -9,14 +9,15 @@ import { type SkillsStepDto } from '~/bundles/talent-onboarding/types/types.js';
 
 import { MAX_LINKS } from '../constants/constants.js';
 import styles from '../styles.module.scss';
+import { CloseIconButton } from './close-icon/close-icon-button.js';
 
 type Properties = {
     control: Control<SkillsStepDto>;
     errors: FieldErrors<SkillsStepDto>;
 };
 
 const SkillsProjectLinks: React.FC<Properties> = ({ control, errors }) => {
-    const { fields, append } = useFieldArray({
+    const { fields, append, remove } = useFieldArray({
```

Comment:
Not related to this pr but I think it will be better to move this hook import to hooks.ts and import it from there

Reply:
Fixed this import